resources/js/lang/en.js

```diff
@@ -48,19 +46,12 @@ export default {
                 1: "While investigating the \"Gloom\", you discovered the bandits from the Black Barrow are associated with some sort of cult. Your arrival disrupted some sort of ritual. Among the scripts found in the ritual chamber are notes on a couple of other places of power in the area. One appears to see heavy use with the cult. It looks as though you have an opportunity to disrupt more of the cult's activities.{br}You followed the writings to an ancient crypt believed to be the cult’s base of operations. A group of cultists were performing ritual incantations in front of a black, gaping hole in reality. With the cultists and their minions dead, the dark rift is now dormant."
             },
         },
-        5: {
```

Comment:
This quest is redundant because quest 9 talks about the same progress. And this quest in only shown on a single scenario where quest 9 was already shown.

Quest 9 shows this summary:
> You have met the Aesther enchantress, Hail, at the Crooked Bone. Although a bit eccentric, she does seem quite capable. You have earned her favor by aiding in her endeavors. Perhaps now she can help you find Jekserah?